src/simulator/AccelerometerModule.tsx

```diff
@@ -113,10 +123,11 @@ const Gesture = ({ icon, state, enabled, onValueChange }: GestureProps) => {
         value={choice}
         onChange={handleSelectChange}
         ref={selectRef}
+        fontFamily="code"
```

Comment:
Agreed this morning that we'd remove this as we can't apply it only to the English text and it doesn't make sense for the translation.

Reply:
[`f6037ec`](https://github.com/microbit-foundation/python-editor-v3/pull/993/commits/f6037ecd0b16471f40d369e3c9e515d5065b4660)